.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "lib/forge-std"]
 	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "scw-contracts"]
+	path = scw-contracts
```

Comment:
all submodules should be a part of the lib folder

Reply:
oh ok.